libraries/cyclestreets-view/src/main/AndroidManifest.xml

```diff
@@ -13,18 +13,26 @@
   <application
       android:requestLegacyExternalStorage="true"
       android:supportsRtl="false">
-    <activity android:name="net.cyclestreets.AccountDetailsActivity"
-              android:label="Account Details"
-              android:windowSoftInputMode="stateVisible|adjustResize" />
-    <activity android:name="net.cyclestreets.FeedbackActivity"
-              android:label="Route Comments"
-              android:windowSoftInputMode="stateVisible|adjustResize" />
-    <activity android:name="net.cyclestreets.LiveRideActivity"
-              android:label="LiveRide"
-              android:launchMode="singleTop"
-              android:screenOrientation="portrait" />
+        <activity
+            android:name="net.cyclestreets.views.CircularRouteActivity"
+            android:theme="@style/Theme.AppCompat.DayNight.DarkActionBar">
```

Comment:
If this _does_ need to be its own activity, why is it themed where others are not? and why is it within `views` package when other activities are not?

Reply:
I wanted to have the more modern theme of the back arrow at the top, which I don't think existed in earlier Android versions.
It wasn't clear to me how the classes are grouped in the packages, so I put it in the same place as the overlays as it's coming from the TapToRouteOverlay - I'm happy to move it to where the other activities are if you prefer.

Reply:
Perhaps you could include some screenshots of the new user journeys etc so I can see what it all looks like without running it locally, and then we can discuss.

Reply:
Apologies, I should have included a link to the issues thread, where I've described the flow with screenshots:
https://github.com/cyclestreets/android/issues/408

Reply:
I've added screenshots for the Distance tab and also for LiveRide / Replan route.